### DISCUSSION TITLE
Scihubs database and filecoin

### DISCUSSION THREAD

**Discussion by mrmola** [filecoin-project/community#188] [General]
Seeing as scihub is going under, I was wondering if it would be possible to use filecoin or something similar to backup scihub's database.

**Comment by mishmosh** [comment]
Batsignaling @dineshshenoy and @jnthnvctr to check out this idea.

**Comment by dineshshenoy** [comment]
This is interesting @mrmola - do you know anyone there where we can facilitate this?

**Comment by bertrandfalguiere** [comment]
A first step could be a collaborative IPFS cluster (collab.ipfs.io) that could then be replicated on Filecoin?
